src/test/java/RestAssuredTests/TestGet.java

```diff
@@ -0,0 +1,45 @@
+package RestAssuredTests;
+
+import APICalls.GetCall;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import utility.PropertyManager;
+
+import static org.hamcrest.Matchers.equalTo;
+
+
+import java.io.IOException;
+
+public class TestGet extends GetCall {
+    public TestGet() throws IOException {
+    }
+
+
+    static RequestSpecification httpRequest = RestAssured.given().when().header("Content-Type", "application/json");
+    static Response response = httpRequest.get(GET_URL);
+
+    @Test
+    public static void testId() throws IOException {
+        response.then().assertThat().body("id", equalTo(PropertyManager.getId()));
+    }
+    @Test
+    public static void testName() throws IOException {
+        response.then().assertThat().body("name", equalTo("test_board"));
+    }
+    @Test
+    public static void checkIfResponseCodeOfGetIsEqualTo200(){
+        int statusCode = response.getStatusCode();
+        Assert.assertEquals(statusCode, 200);
```

Comment:
there is a mixed use of RestAssured and TestNG assertions
consistency in assertion styles is generally a good practice
if you're using RestAssured's fluent assertions, consider using them throughout for consistency